### PR TITLE
[LOOP-2411] Onboarding How the App Works

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-2411
- Add How the App Works Onboarding section
- Set all existing PDF assets to universal
- Add missing pre-meal and workout images to demo
- Minor code updates

### CODE REVIEW

**Comment by darinkrauss** [comment]
LW PR for build: https://github.com/tidepool-org/LoopWorkspace/pull/473

File: TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```
Comment (nhamming) [inline TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: [nit] I'm still confused by all this `1x`, `2x`, and `3x`, given the contents for `HowTheAppWorks_10_1.imageset` above.

Comment (nhamming) [reply TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: [nit] I see `.png` are being added for `2x`. I thought images should be `.pdf`?

**Review by nhamming — APPROVED** [review]
Just a question about `.png` and `.pdf`. LGTM

File: TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json (hunk 1 above)
Comment (ps2) [reply TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: I do wonder if using single scale is better for these assets.

**Review by ps2 — APPROVED** [review]
LGTM; just a comment on multiple scales.

File: TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json (hunk 1 above)
Comment (darinkrauss) [reply TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: I was explicitly instructed by the design team to use PDFs for the all-vector graphics and PNGs for the screenshots.

I'm not sure what you are asking me to do here.

Do we all agree that the all-vector graphics should be PDF set as "Single Scale" in Xcode? I think that is the only thing that makes sense from both a scalability and disk size perspective.

Regarding the screenshot PNGs, are you suggesting to:
1. change them all to "Single Scale" in Xcode, or
2. change them all into PDFs set as "Single Scale" in Xcode, or
3. something else?

Please let me know. Thanks!

Comment (darinkrauss) [reply TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: @nhamming @ps2 I'd really like to resolve these concerns you both have so they won't come up in future onboarding work. Any clarification is greatly appreciated. Thanks!

Comment (nhamming) [reply TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json:20]: I remember a discussion within the devs to only use `.pdf`s and this is where my comment is coming from. Curious why design wants `.png` instead.

**Pull request merged**